client/jetpack-cloud/sections/partner-portal/primary/wpcom-atomic-hosting/card-content.tsx

```diff
@@ -0,0 +1,132 @@
+import { Button, JetpackLogo, WooLogo, CloudLogo } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import Tooltip from 'calypso/components/tooltip';
+
+import './style.scss';
+
+// FIXME: These should be imported from a shared file when available
+const JETPACK_HOSTING_WPCOM_BUSINESS = 'JETPACK_HOSTING_WPCOM_BUSINESS';
+const JETPACK_HOSTING_WPCOM_ECOMMERCE = 'JETPACK_HOSTING_WPCOM_ECOMMERCE';
+
+interface PlanInfo {
+	title: string;
+	description: string;
+	price: string;
+	interval: string;
+	wpcomFeatures: string[];
+	jetpackFeatures: string[];
+	storage: string;
+	logo: JSX.Element | null;
+}
+
+export default function CardContent( { planSlug }: { planSlug: string } ) {
+	const translate = useTranslate();
+	const tooltipRef = useRef< HTMLDivElement | null >( null );
+	const [ showPopover, setShowPopover ] = useState( false );
+
+	const getLogo = ( planSlug: string ) => {
```

Comment:
Non-blocking:

If we're simply mapping from one value to another, an alternative to a function and a `switch-case` block would be an object:

```js
const LOGOS = {
	[ JETPACK_HOSTING_WPCOM_BUSINESS ]: <CloudLogo />,
	[ JETPACK_HOSTING_WPCOM_ECOMMERCE ]: <WooLogo />,
};

const myLog = LOGOS[ planSlug ]; // automatically returns null if not found
```

Reply:
Thanks for your review, @mattgawarecki. This is a good approach. 

This method might change based on the API response. So, I would like to keep this as it is for now. 

> Though, on second thought, this approach renders two components every time. 🤷

Could you please approve this PR if everything else looks good?